bolt/test/AArch64/test-indirect-branch.s

```diff
@@ -0,0 +1,110 @@
+// Test how BOLT handles indirect branch sequence of instructions in
+// AArch64MCPlus builder.
+// This test checks the pattern where there is no shift amount after add
```

Comment:
nit: replace:
```cpp
// This test checks the pattern where there is no shift amount after add
// instruction. The pattern come from libc, it can be reproduced with
// a 'static' built binary.
```

with something like (first line is an empty comment to separate from above):
```cpp
//
// Pattern 1: there is no shift amount after the 'add' instruction.

```

Reply:
Done.